Siena-MongoDB implementation

thagikura opened the pull request:
I implemented Siena-MongoDB.

I didn't know there is already mongodb branch in Siena.
If there is no problem, please try it.

thagikura committed:
Add MonngoDBPersistenceManager

thagikura committed:
add mongo-driver dependencies

mandubian commented:
Hi Takeshi!

This is really great! Thanks a lot!
I'm going to investigate your code!
My mongo branch provided already a few features but not all.
I had also rewritten lots of unit tests for Mongo...

Did you implement everything or just a part of it?

Best regards
Pascal

thagikura commented:
Hi Pascal!
Thank you for your reply.

Actually I implemented part of it. Those source codes are based on that post on Google Code about a year ago.
( http://code.google.com/p/siena/issues/detail?id=6 )

But that codes didn't work under the current version of Siena APIs.
So I fixed and added JUnit test codes to it.

Please investigate it, thanks.

Best regards
Takeshi

mandubian commented:
I'm going to fetch your fork directly and work from it because I don't want to merge your pull request into the trunk but I'd prefer working on the mongodb branch!

thagikura commented:
Ok. thanks!

thagikura commented:
Hi Pascal,

How is the investigation going?
If there is a slightest thing to ask me, please let me know, thanks!

mandubian commented:
I'm late :D
Sorry but I had no time to work on it!

Pascal

On Sun, Nov 27, 2011 at 7:56 AM, Takeshi Hagikura <
reply@reply.github.com

> wrote:
> 
> Hi Pascal,
> 
> How is the investigation going?
> If there is a slightest thing to ask me, please let me know, thanks!
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/mandubian/siena/pull/8#issuecomment-2886311

mrcritical commented:
Still waiting for Mongo support.  I really don't like Morphia, which is the only other real option for Play!.

adpande commented:
Any updates on mongodb support?

Regards,
